frontend/site/.vitepress/theme/components/Footer.vue

```diff
@@ -13,7 +13,7 @@
         <div class="fr-footer__content">
           <p class="fr-footer__content-desc">
             Le code source est ouvert et les contributions sont bienvenues.
-            <a href="https://github.com/etalab/suivi-ouverture/"
+            <a href="https://github.com/etalab/ouverture.data.gouv.fr"
```

Comment:
I'm adding this comment here, but the .md pages are no longer working, /accessibilite for example.

Reply:
Fixed